inc/admin/metabox/src/components/MetaFieldsManager.js

```diff
@@ -0,0 +1,70 @@
+/* global metaSidebar */
+import { PanelBody } from "@wordpress/components";
```

Comment:
Dependencies from `@wordpress`  can be used directly from core:

`const { PanelBody } = wp.components;`